miasm2/ir/symbexec.py

```diff
@@ -442,6 +442,7 @@ def emulbloc(self, irb, step=False):
         """
         for assignblk in irb.irs:
             if step:
+                print 'Instr', assignblk.instr
```

Comment:
`.instr` is not necessarily defined. Do you prefer to display a `Instr: None`, or nothing?